packages/@aws-cdk/aws-servicecatalogappregistry/lib/attribute-group.ts

```diff
@@ -0,0 +1,113 @@
+import * as cdk from '@aws-cdk/core';
+import { InputValidator } from './private/validation';
+import { CfnAttributeGroup } from './servicecatalogappregistry.generated';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct } from 'constructs';
+
+/**
+ * A Service Catalog AppRegistry Attribute Group.
+ */
+export interface IAttributeGroup extends cdk.IResource {
+  /**
+   * The ARN of the attribute group.
+   * @attribute
+   */
+  readonly attributeGroupArn: string;
+
+  /**
+   * The ID of the attribute group.
+   * @attribute
+   */
+  readonly attributeGroupId: string;
+}
+
+/**
+ * Attributes for the attribute group, defined by a nested list of
+ * key-value pairs.
+ */
+export interface Attribute {
+  [key: string]: string | number | boolean | Attribute |
+  string[] | number[] | boolean[] | Attribute[];
+}
```

Comment:
I don't think this adds much. I would remove this type, and declare `attributes` in `AttributeGroupProps `as

```ts
readonly attributes?: { [key: string]: any };
```

Reply:
Okay, my understanding that having `any` was frowned upon for typing.